### PR TITLE
RELATED: RAIL-1092 Describe data used in examples

### PR DESCRIPTION
**Pull request by ocetnik**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1322/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/824/ : FAILURE in 3m 24s

File: examples/src/routes/AboutThisProject.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
+// (C) 2007-2018 GoodData Corporation
+import React from 'react';
+
+const logicalDataModel = require('../static/logicalDataModel.png');
+
+export const AboutThisProject = () => (
+    <div>
+        <h1>About This Project</h1>
+        <p>
+            This project (ID <code>xms7ga4tf3g3nzucd8380o2bev8oeknp</code>) is created specifically for testing
```
Comment (tomasreichmann) [inline examples/src/routes/AboutThisProject.jsx]: import { projectId } from '...utils/fixtures'

**Comment by tomasreichmann** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1326/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/827/ : SUCCESS in 5m 40s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/273/ : SUCCESS in 1m 00s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/829/ : FAILURE in 5m 57s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/460/ : SUCCESS in 10m 44s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/176/ : SUCCESS in 11m 52s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/274/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/830/ : SUCCESS in 1m 30s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/461/ : SUCCESS in 8m 21s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/177/ : SUCCESS in 7m 13s

**Pull request merged**